TestVector_Types/double hangs on Cassandra 5-LATEST causing CI timeout

## Summary

`TestVector_Types/double` hangs indefinitely when running against Cassandra 5-LATEST (5.0.6), causing the entire integration test suite to hit the 5-minute timeout.

## CI Evidence

- Failed run: https://github.com/scylladb/gocql/actions/runs/22770210966/job/66048942765
- The test starts but never completes, blocking all subsequent tests

## Timeout panic output

```
panic: test timed out after 5m0s
	running tests:
		TestVector_Types (10s)
		TestVector_Types/double (0s)
```

The goroutine dump shows the test is stuck waiting on a TLS read (`crypto/tls.(*Conn).readRecordOrCCS`), suggesting the connection to Cassandra hangs during the vector type test for `double`.

## Impact

This blocks the Cassandra 5-LATEST CI job on every PR. All other jobs (Scylla LATEST/LTS-LATEST/LTS-PRIOR/PRIOR, Cassandra 4-LATEST) pass.

## Relevant code

- `vector_test.go:102` — `TestVector_Types`

--- a/Makefile
+++ b/Makefile
@@ -205,6 +205,9 @@ cassandra-start: .prepare-pki .prepare-cassandra-ccm .prepare-java resolve-cassa
 	ccm remove ${CCM_CASSANDRA_CLUSTER_NAME} 2>/dev/null 1>&2 || true
 	ccm create ${CCM_CASSANDRA_CLUSTER_NAME} -i ${CCM_CASSANDRA_IP_PREFIX} -v "$${CASSANDRA_VERSION_RESOLVED}" -n3 -d --vnodes --jvm_arg="-Xmx256m -XX:NewSize=100m"
 	ccm updateconf ${CASSANDRA_CONFIG}
+	for conf_dir in ${CCM_CONFIG_DIR}/${CCM_CASSANDRA_CLUSTER_NAME}/node*/conf; do \
+		sed -i 's/^#MAX_HEAP_SIZE=.*/MAX_HEAP_SIZE="256M"/' "$$conf_dir/cassandra-env.sh"; \
+	done
 	ccm start --wait-for-binary-proto --wait-other-notice --verbose
 	ccm status
 	ccm node1 nodetool status
@@ -282,8 +285,8 @@ test-integration-cassandra: cassandra-start
 		echo "Cassandra version ${CASSANDRA_VERSION} was not resolved"
 		exit 1
 	fi
-	echo "go test -v ${TEST_OPTS} -tags \"${TEST_INTEGRATION_TAGS}\" -distribution cassandra -timeout=5m -runauth -gocql.timeout=60s -runssl -proto=${TEST_CQL_PROTOCOL} -rf=3 -clusterSize=3 -autowait=2000ms -compressor=${TEST_COMPRESSOR} -gocql.cversion=$${CASSANDRA_VERSION_RESOLVED} -cluster=$$(ccm liveset) ./..."
-	go test -v ${TEST_OPTS} -tags "${TEST_INTEGRATION_TAGS}" -distribution cassandra -timeout=5m -runauth -gocql.timeout=60s -runssl -proto=${TEST_CQL_PROTOCOL} -rf=3 -clusterSize=3 -autowait=2000ms -compressor=${TEST_COMPRESSOR} -gocql.cversion=$$(ccm node1 versionfrombuild) -cluster=$$(ccm liveset) ./...
+	echo "go test -v ${TEST_OPTS} -tags \"${TEST_INTEGRATION_TAGS}\" -distribution cassandra -timeout=10m -runauth -gocql.timeout=60s -runssl -proto=${TEST_CQL_PROTOCOL} -rf=3 -clusterSize=3 -autowait=2000ms -compressor=${TEST_COMPRESSOR} -gocql.cversion=$${CASSANDRA_VERSION_RESOLVED} -cluster=$$(ccm liveset) ./..."
+	go test -v ${TEST_OPTS} -tags "${TEST_INTEGRATION_TAGS}" -distribution cassandra -timeout=10m -runauth -gocql.timeout=60s -runssl -proto=${TEST_CQL_PROTOCOL} -rf=3 -clusterSize=3 -autowait=2000ms -compressor=${TEST_COMPRESSOR} -gocql.cversion=$$(ccm node1 versionfrombuild) -cluster=$$(ccm liveset) ./...
 
 test-integration-scylla: scylla-start
 	@echo "Run integration tests for proto ${TEST_CQL_PROTOCOL} on scylla ${SCYLLA_VERSION}"

